apps/web/pages/settings/organizations/[id]/add-teams.tsx

```diff
@@ -0,0 +1,43 @@
+import Head from "next/head";
+import { useRouter } from "next/router";
+
+import { AddNewTeamsForm } from "@calcom/features/ee/organizations/components";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { WizardLayout } from "@calcom/ui";
+
+import PageWrapper from "@components/PageWrapper";
+
+const AddNewTeamsPage = () => {
+  const { t } = useLocale();
+  const router = useRouter();
+  if (!router.isReady) return null;
+  return (
+    <>
+      <Head>
+        <title>Create your teams</title>
```

Comment:
NIT: needs to be i18n string

Reply:
we also have a <Meta> component

Reply:
Resolved in the [correct PR](https://github.com/calcom/cal.com/pull/9184) for these changes, thanks!